tests/Unit/inc/ThirdParty/Themes/Divi/disableImageDimensionsHeightPercentage.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace WP_Rocket\Tests\Unit\inc\ThirdParty\Themes\Divi;
+
+use Mockery;
+use WP_Rocket\Tests\Unit\TestCase;
+use Brain\Monkey\Functions;
+use WP_Rocket\ThirdParty\Themes\Divi;
+use WP_Theme;
+
+/**
+ * @covers \WP_Rocket\ThirdParty\Themes\Divi::disable_image_dimensions_height_percentage
+ * @uses   ::is_divi
+ *
+ * @group  ThirdParty
+ */
+class Test_DisableImageDimensionsHeightPercentage extends TestCase {
+	/**
+	 * @dataProvider providerTestData
+	 */
+	public function testAddDiviToDescription( $config, $expected ) {
```

Comment:
where is integration test, I think we need it and keep in mind that we need to save the current theme at setup and then revert it back at tearDown

Reply:
Done... The WPThemeTestcase we're extending takes care of setting theme at setup and teardown. :)